### ISSUE TITLE
Sandwiches contain constant full or partial order

### ISSUE THREAD

**Issue by calbers1** [saladtheory/saladtheory.github.io#33]
While I am thrilled to find my position on salad is MOSTLY validated by other possibly insane people, I have one issue. Salads, in my opinion, are always characterized by either a distinct lack of order, which would exclude many foods that always follow a full or partial pattern, i.e. a ham and cheese sandwich.

My position is that we can use a visual model to describe the difference in a very easy-to-see way. 
If we represent each individual ingredient as a differently colored cube or pixel, and we take slices across the sample on a single axis to represent the food (ideally whichever axis will give us the largest quantity of ingredients per slice), then we can compare the slices to determine "salidity". If the slices show no uniformity of pattern, it is a salad. If the slices show a distinct order (even if partial), it is a sandwich. 

Slices across a ham sandwich will always be similar if not identical. Slices across a bowl of tortilla soup will almost never be identical.
slices across a chicken salad sandwich will not be identical, but will always have bread structured on the outside, and would therefore have enough consistent structure to be a sandwich. 

I have yet to find any food that does not conform to one of these two archetypes; however, I see a VERY distinct split between the two when I examine a large quantity of different foods.

**Comment by gmalmquist** [comment]
This is interesting! Do you have a way of formalizing a formula for that uniformity? Off the cuff, it sounds it would be tricky without pulling in some computer vision algorithms, and that doesn't make for a straightforward definition.

**Comment by amitai** [comment]
> Salads, in my opinion, are always characterized by either a distinct lack of order, which would exclude many foods that always follow a full or partial pattern, 

I disagree. Caprese salad follows a pattern, as you can see in any image search.
